shielddb-gson-backend/src/test/java/nf/fr/k49/shielddb/gson/BottomShieldTest.java

```diff
@@ -0,0 +1,68 @@
+package nf.fr.k49.shielddb.gson;
+
+import nf.fr.k49.shielddb.core.shield.BottomShield;
+import nf.fr.k49.shielddb.core.storage.FileStorage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.spy;
+
+public class BottomShieldTest {
+
+    private static final String BASE_DIR = "target/manual";
+
+    private FileStorage storage;
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Before
+    public void setUp() throws IOException {
+        String storagePath = BASE_DIR + "/User.json";
+
+        storage = spy(new FileStorage(storagePath));
+
+        String json = "[ \"String1\", \"String2\", \"String3\" ]";
+
+        storage.write(json);
```

Comment:
why did you add blank lines between every line of this method?

Reply:
For me we should add a blank line between code portion that are logically distinct. In my mind I think if we add a blank line maybe we just need to create a function/method. In test case, I add a blank line between the 3 steps (given / when / then) of the test case but that all. Here in the setup you just make "storage initialization" so it's one thing to do, so yeah I think it's more readable to remove these blank lines